examples/python_scripting.py

```diff
@@ -77,13 +75,12 @@ def do_cd(self, arglist):
             except Exception as ex:
                 err = '{}'.format(ex)
             else:
-                out = 'Successfully changed directory to {!r}\n'.format(path)
-                self.stdout.write(out)
+                self.poutput('Successfully changed directory to {!r}\n'.format(path))
```

Comment:
You can remove `\n` since poutput() already adds one.

Reply:
Removed that. Also changed everything to in the file to use f-strings.

Reply:
We have to support Python 3.5. Our examples can't have f-strings, as nice as they are.

Reply:
As long as it is only in example code, I wouldn't strongly object to begin using f-strings.  But if we do that, I would want at nice warning in the docstring at the top of the file saying something like **WARNING: This example requires Python 3.6+***